site/content/blog/2023-12-18-user-personas-poll/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Discovering Our Users - The ZAP User Personas Questionnaire"
+summary: >
+  Join our journey to tailor ZAP for every user, by sharing your unique insights and experiences. Your perspective is the key to unlocking ZAP's full potential for everyone in the cybersecurity community.
+images:
+- https://www.zaproxy.org/blog/2023-12-18-user-personas-poll/images/535a4e71-d6a6-40a1-8289-f31e8810b65b.png
+type: post
+tags:
+- blog
+- questionnaire 
+date: "2023-12-20"
+authors:
+- yiannis
+---
+
+The Zed Attack Proxy (ZAP) team is dedicated to making our tool the best it can be for our diverse user base. As the cybersecurity landscape evolves, so do the needs of our users. That's why we're excited to announce the **ZAP User Personas Questionnaire** — an initiative to better understand and cater to the unique requirements of everyone who uses ZAP.
+
+### **Why This Matters**
```

Comment:
Headigs shouldn't also need bold tags. (Throughout)

Reply:
This is an old blog post, added by accident? https://www.zaproxy.org/blog/2023-12-21-user-personas-poll/